### PR TITLE
Game balance sept15

### PR DESCRIPTION
**Pull request by Phrancis**
Game balance adjustments, need to inspect & test server build so it is safe to merge. At the time it had some build issues with Travis though I'm pretty sure it built fine at least on my machine.

### CODE REVIEW

**Comment by marcandregirard** [comment]
https://github.com/Cardshifter/Cardshifter/blob/game-balance-sept15/extra-resources/mods/Cyborg-Chronicles/test.groovy  => You have a failing test right here 

```
from clearState test 'heal creatures at end of turn' using {
    def attacker = to you zone 'Battlefield' create {
        creature 'Mech'
        attack 2
        health 2
    }
    def defender = to opponent zone 'Battlefield' create {
        creature 'Mech'
        attack 1
        health 4
    }

    uses 'End Turn' ok
    uses 'End Turn' ok
    uses 'Attack' on attacker withTarget defender ok
    assert attacker.health == 1
    assert defender.health == 2
    uses 'End Turn' ok
    assert attacker.health == 2
    assert defender.health == 4
}
```

> assert attacker.health == 2
>                 |        |      |
>                 |        1      false

File: extra-resources/mods/Cyborg-Chronicles/Game.groovy [hunk 1]
```diff
@@ -258,17 +263,6 @@ rules {
             // ...return from this method without doing anything
             return
         }
-        // CYBORG-CHRONICLES MECHANIC:
```
Comment (marcandregirard) [inline extra-resources/mods/Cyborg-Chronicles/Game.groovy]: This removal is probably what is causing the test failing

Comment (Phrancis) [inline extra-resources/mods/Cyborg-Chronicles/Game.groovy]: I don't remember why I deleted that, come to think of it. I'll look for a commit message. Could you try adding it back in and try to run the tests?

**Comment by Zomis** [comment]
There's just one thing that makes me hesitant to merge this now, and that's the change in creature name from Upgrado to Konstruktor - this means that some saved decks will break. We should probably either rename it back to Upgrado or set a specific Id on the card to match the old auto-generated id ("upgrado-mk-i" for example).

**Comment by Phrancis** [comment]
Let's not rename it for now, Upgrado is a fine name.

**Pull request merged**